frontend/server/k8s-helper.ts

```diff
@@ -70,29 +94,7 @@ export async function newTensorboardInstance(logdir: string): Promise<void> {
       tensorboardSpec: {
         logDir: logdir,
       },
-      podTemplateSpec: {
-        spec: {
-          containers: [{
-            env: [{
-              name: "GOOGLE_APPLICATION_CREDENTIALS",
-              value: "/secret/gcp-credentials/user-gcp-sa.json"
-            }],
-            volumeMounts: [{
-              name: "gcp-credentials",
-              mountPath: "/secret/gcp-credentials/user-gcp-sa.json",
-              readOnly: true
-            }]
-          }],
-          volumes: [{
-            name: "gcp-credentials",
-            volumeSource: {
-              secret: {
-                secretName: "user-gcp-sa"
-              }
-            }
-          }]
-        }
-      }
+      podTemplateSpec
```

Comment:
@eterna2 @rileyjbauer : pardon me if this is trivial (I don't code in ts or Go), but shouldn't this need a json key podTemplateSpec as well?

The default defaultPodTemplateSpec above returns a {spec: {...}} json but as per the samples tensorboard-local-volume.yaml there should be a dict key podTemplateSpec that wraps the actual spec.

Reply:
for es6 (Javascript), you don't need to provide a key if the var name is same as the key.

i.e.
```
var foo = 'bar';

{ foo } is equivalent to { foo: foo }
```